typescript/libs/node_client.py

```diff
@@ -252,7 +257,7 @@ def __init__(self, script_path):
             self.server_proc = None
         else:
             global_vars._node_path = node_path
-            print("Found node executable at " + node_path)
+            print("Try spawning node executable from: " + node_path)
```

Comment:
Remove "Try" (or change it to "Trying to spawn") - rigth now it suggests that the user needs to do something special.